### DISCUSSION TITLE
Multiple SDRs running at the same time feeding rtlamr?

### DISCUSSION THREAD

**Discussion by rocketman889** [bemasher/rtlamr#283] [General]
Good Day All,

First off, thanks bemasher for this wonderful code! It really has made life better and easier for me!

How can I run multiple SDRs using different rtl_tcp sessions (in different terminal windows) and have them all update the rtlamr service/code? Do I need to be running multiple rtlamr sessions and if so, how do I point them to the different rtl_tcp ip/port addresses?

Thanks!

**Comment by bemasher** [comment]
Use listen port `-p` and device index `-d` flags on `rtl_tcp`. In `rtlamr` use the `-server` flag to specify host and port.

**Reply by rocketman889** [reply]
Thanks for your feedback! I was able to get two instances working.

A follow-up question, similar to another discussion around the scanning of the entire frequency range/channels #261. You mentioned it depends on a couple of factors, but as long as that meter is hopping with the channel(s) covered by the bandwidth, then rtlamr should be able to decode it?

My SDR (when running in GQRX and centered at 912600115) will show from ~911.7 MHz - 913.5 MHz, which looking at the datasheet covers 7 channels. So if the meter is hopping all through the channels, then having additional SDRs centered at other frequencies should help capture messages?

![2023-08-08_17h29_57](https://github.com/bemasher/rtlamr/assets/965462/771938f0-ddd1-4fa9-83fe-46009e483e4b)

I also had a LNA with a DC block. Is keeping the gain at default in the rtl_tcp (0 = automatic) the best or should I set it based on what seems to be the best from the GQRX and playing around with the settings?
![2023-08-16_17h26_41](https://github.com/bemasher/rtlamr/assets/965462/ef34ebe2-806a-46d7-aa27-ad18019383aa)
